plugin/src/main/groovy/com/novoda/staticanalysis/internal/detekt/CollectDetektViolationsTask.groovy

```diff
@@ -1,17 +0,0 @@
-package com.novoda.staticanalysis.internal.detekt
-
-import com.novoda.staticanalysis.Violations
-import com.novoda.staticanalysis.internal.CollectViolationsTask
-import groovy.util.slurpersupport.GPathResult
-
-class CollectDetektViolationsTask extends CollectViolationsTask {
```

Comment:
Deleted since Detekt uses Checkstyle format and I don't think they will ever change that.